docs/framework/wpf/graphics-multimedia/graphics-rendering-registry-settings.md

```diff
@@ -72,7 +72,7 @@ This topic provides an overview of the [!INCLUDE[TLA2#tla_winclient](../../../..
   
  The required video driver setting takes a string of the following format:  
   
-||  
+| |  
```

Comment:
I'm not sure that the table is even needed here, since it's a single cell. That's also true of the next file. Alternately, it's not celar to me that we need to do anything here : the table looks OK in the original Markdig-published page: https://review.docs.microsoft.com/en-us/dotnet/framework/wpf/graphics-multimedia/graphics-rendering-registry-settings?branch=markdig-migration

Reply:
It sounded like the spreadsheet was complaining about that missing space. Thought it's tough to determine the exact problem from the sheet.